### PR TITLE
refactor!: Rework Command via MessageBus for new Request API response topic

### PR DESCRIPTION
**Pull request by lenny-intel**
BREAKING CHANGE: Command via MessageBus Topic configurations have changed

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A**
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?) **N/A**
  <link to docs PR>

## Testing Instructions
See https://github.com/edgexfoundry/edgex-go/pull/4310

### CODE REVIEW

File: internal/controller/messaging/command.go [hunk 1]
```diff
@@ -25,15 +25,14 @@ import (
 )
 
 const (
-	CommandRequestTopic        = "CommandRequestTopic"
-	CommandResponseTopicPrefix = "CommandResponseTopicPrefix"
+	CommandRequestTopic = "CommandRequestTopic"
```
Comment (jim-wang-intel) [inline internal/controller/messaging/command.go:28]: do we have the constant from go.mod.contract?

Comment (lenny-intel) [reply internal/controller/messaging/command.go:28]: Good catch. Will update to use from core contracts.

Comment (lenny-intel) [reply internal/controller/messaging/command.go]: Actually, was already using one from core contracts, but missed removing this one. Removed now.

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by ejlee3 — APPROVED** [review]
LGTM!

**Pull request merged**